code/game/objects/structures/watercloset.dm

```diff
@@ -109,7 +109,7 @@
 	density = 0
 	anchored = 1
 
-/obj/structure/urinal/attackby(obj/item/I as obj, mob/user as mob)
+/obj/structure/urinal/attackby(obj/item/I as obj, mob/user as mob, var/damtype = BRUTE)
```

Comment:
```suggestion
/obj/structure/urinal/attackby(obj/item/I, mob/user)
```

Reply:
Oh woops, forgot we defined that in apply_damage()